Examples/UtilityRateForecastExample.py

```diff
@@ -0,0 +1,83 @@
+import json
+import os
+import requests
+import numpy as np
+import pandas as pd
+
+import PySAM.Utilityrateforecast as utility_rate_forecast
+import PySAM.ResourceTools
+
```

Comment:
I think it would be really helpful if there was a short intro paragraph explaining what this is doing